deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/amqp/AmqpDeviceOperations.java

```diff
@@ -168,9 +168,4 @@ protected void addAmqpLinkProperty(String key, String value)
     {
         this.amqpProperties.put(Symbol.getSymbol(key), value);
     }
-
-    protected void clearAmqpLinkProperty()
```

Comment:
Unused in our code, and inaccessible to users of this library, so it is safe to remove